build/index_core_compatible.js

```diff
@@ -1,20 +1,37 @@
-import Experiment from '../es6/experiment';
-import Interpreter from '../es6/interpreter';
-import RandomPlanoutCoreCompatible from '../es6/ops/randomPlanoutCoreCompatible';
-import Core from '../es6/ops/core';
-import * as Namespace from '../es6/namespace';
 import Assignment from '../es6/assignment';
+import Experiment from '../es6/experiment';
 import ExperimentSetup from '../es6/experimentSetup';
+import Interpreter from '../es6/interpreter';
+import * as LibUtils from '../es6/lib/utils';
+import Random from '../es6/ops/randomPlanoutCoreCompatible';
+import * as Base from '../es6/ops/base';
+import * as Core from '../es6/ops/core';
+import * as OpsUtils from '../es6/ops/utils';
+import * as Namespace from '../es6/namespacePlanoutCoreCompatible';
+
+OpsUtils.registerOperators({
+    "randomFloat": Random.RandomFloat,
+    "randomInteger": Random.RandomInteger,
+    "bernoulliTrial": Random.BernoulliTrial,
+    "bernoulliFilter": Random.BernoulliFilter,
+    "uniformChoice": Random.UniformChoice,
+    "weightedChoice": Random.WeightedChoice,
+    "sample": Random.Sample
+}, true);
```

Comment:
I think we can get by. In the mean time, I'm going to push up my attempt at doing your suggestions above. At the very least, it might help inform your take on it.